bridge/opencensus/bridge_test.go

```diff
@@ -0,0 +1,280 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opencensus
+
+import (
+	"context"
+	"testing"
+
+	octrace "go.opencensus.io/trace"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/global"
+	"go.opentelemetry.io/otel/label"
+	"go.opentelemetry.io/otel/oteltest"
+)
+
+func TestMixedAPIs(t *testing.T) {
+	sr := new(oteltest.StandardSpanRecorder)
+	tp := oteltest.NewTracerProvider(oteltest.WithSpanRecorder(sr))
+	global.SetTracerProvider(tp)
```

Comment:
I'm not 100%, but it looks like we don't need to register this TracerProvider with the global package as it is directly used. This would also apply to the other test functions.

Reply:
Good call.  Removed here and elsewhere.